server/src/message/data.ts

```diff
@@ -5,77 +5,26 @@ export const MESSAGES: WeatherMessage[] = [
     id: '1',
     rain_intensity: 'intense',
     message:
-      'Cuidado {when}, o mundo irá cair! Leve um guarda-chuva, mas talvez precise de um barco',
+      'Did you rent a boat? Maybe could be a good idea, a river will fall from the sky today',
   },
   {
     id: '2',
-    rain_intensity: 'intense',
-    message: 'Pronto para um dilúvio? {when} tem tempestade chegando',
-  },
-  {
-    id: '3',
-    rain_intensity: 'intense',
-    message:
-      'F*deu, espero que tenha um guarda-chuva contigo, {when} vai chover como nunca antes',
-  },
-  {
-    id: '4',
     rain_intensity: 'high',
-    message: 'Chuva high {when}',
-  },
-  {
-    id: '5',
-    rain_intensity: 'high',
-    message: 'Chuva high {when}',
-  },
-  {
-    id: '6',
-    rain_intensity: 'high',
-    message: 'Chuva high {when}',
-  },
-  {
-    id: '7',
-    rain_intensity: 'normal',
-    message: 'Chuva normal {when}',
+    message: 'A raincoat may be needed. It will rain a lot',
   },
   {
-    id: '8',
-    rain_intensity: 'normal',
-    message: 'Chuva normal {when}',
-  },
-  {
-    id: '9',
+    id: '3',
     rain_intensity: 'normal',
-    message: 'Chuva normal {when}',
-  },
-  {
-    id: '10',
-    rain_intensity: 'low',
-    message: 'Chuva low {when}',
-  },
-  {
-    id: '11',
-    rain_intensity: 'low',
-    message: 'Chuva low {when}',
+    message: 'You should get an umbrella, it will rain, but not too much',
   },
   {
-    id: '12',
+    id: '4',
     rain_intensity: 'low',
-    message: 'Chuva low {when}',
-  },
-  {
-    id: '13',
-    rain_intensity: 'none',
-    message: 'Sem chuva {when}',
+    message: 'It will just drizzling today, i think an umbrella is not needed',
```

Comment:
I think for now it's better not add first person point-of-view to the messages. I would go from:
> I think an umbrella is not needed

to:
> an umbrella is not needed

or:
> you can leave your umbrella at home